alacritty/src/display.rs

```diff
@@ -192,20 +201,28 @@ impl Display {
         // Update OpenGL projection
         renderer.resize(&size_info);
 
-        // Clear screen
+        // We should call `clear` when window is offscreen, so when `window.show()` happens it
+        // would be with background color instead of uninitialized surface.
```

Comment:
I'm not sure what you mean by this? Are you saying this is an outstanding TODO for Wayland? Or are you documenting the existing code?

I'd assume you're documenting the existing code, so I propose some minor changes to make that more clear:

```suggestion
        // Call `clear` before showing the window, to make sure the surface is initialized.
```